hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -156,15 +160,18 @@ public static long totalNewMessages(OffsetRange[] ranges) {
   private final HashMap<String, Object> kafkaParams;
   private final TypedProperties props;
   protected final String topicName;
+  private final KafkaResetOffsetStrategies autoResetValue;
 
   public KafkaOffsetGen(TypedProperties props) {
     this.props = props;
+
     kafkaParams = new HashMap<>();
     for (Object prop : props.keySet()) {
       kafkaParams.put(prop.toString(), props.get(prop.toString()));
     }
     DataSourceUtils.checkRequiredProperties(props, Collections.singletonList(Config.KAFKA_TOPIC_NAME));
     topicName = props.getString(Config.KAFKA_TOPIC_NAME);
+    autoResetValue = KafkaResetOffsetStrategies.valueOf(props.getString("auto.offset.reset", Config.DEFAULT_AUTO_RESET_OFFSET.toString()).toUpperCase());
```

Comment:
I suggest check the string value of  `auto.offset.reset` and set default value to `LATEST`  if it is not set to `LATEST` or `EARLIEST` before we get the `KafkaResetOffsetStrategies`. 
Because `KafkaResetOffsetStrategies` has only two values: `LATEST` and `LATEST`, if the user set it to other value this line will throw an exception.

the lines bellow is useless:
````
default:
            throw new HoodieNotSupportedException("Auto reset value must be one of 'earliest' or 'latest' ");
````

Reply:
@liujinhui1994 any response?